ui-manchette/src/styles/waypoint.css

```diff
@@ -81,12 +80,22 @@
     }
   }
 
+  .menu-wrapper {
+    position: absolute;
+    z-index: 12;
+  }
+}
+
+.last-separator {
+  width: 26px;
+  height: 2rem;
```

Comment:
What is the purpose of this?

Reply:
Was needed when removing the position relative for the waypoint. It is not needed anymore with the new logic and has been reverted.